scripts/rev.php

```diff
@@ -447,12 +448,20 @@ function captureGitValues( & $output )
         if ( trim( $line ) == "" )
             continue;
         if ( substr( $line , 0 , 4 ) == '    ' )
+        {
+            if ( stristr( $line, '[skip-revcheck]' ) !== false )
+            {
+                $skipThisCommit = true;
```

Comment:
This looks like it skips all commits after the first one containing the `[skip-revcheck]` text.
If that is not the case, when does `$skipThisCommit` get set back to `false`?

Reply:
Thank you for the review // fixed at 1f49aff89f4f12fe4c0f995aecf9e20e32eba523